client/memory.js

```diff
@@ -68,225 +77,104 @@ export default class Memory extends React.Component {
       headers: {
         'Authorization': 'Bearer ' + token
       }
-    }).then(function(memory) {
-      // var analyses = JSON.parse(memory['_bodyInit']).tags;
-      var analyses = JSON.parse(memory['_bodyInit']).analyses[0].tags;
-      context.setState({tags: analyses, status: 'Tags:'});
+    }).then(function(res) {
+      var memory = JSON.parse(res['_bodyInit']);
+      var microsoftTags = [];
+      var clarifaiTags = [];
+      if (memory.analyses[0] === null) {
```

Comment:
the nesting of the data changed when adding the microsoft api request. so that's why i wanted to clear the database since old pictures don't contain the same structure of data as the new pictures. i also wasn't sure what the data looks like if either clarifae or microsoft error out and don't send data when analyzing a picture